diff --git a/Classes/TYPO3/Party/Domain/Model/AbstractParty.php b/Classes/TYPO3/Party/Domain/Model/AbstractParty.php
index 6adb9f4..0b91e3a 100644
--- a/Classes/TYPO3/Party/Domain/Model/AbstractParty.php
+++ b/Classes/TYPO3/Party/Domain/Model/AbstractParty.php
@@ -24,7 +24,7 @@ abstract class AbstractParty {
 
 	/**
 	 * @var \Doctrine\Common\Collections\Collection<\TYPO3\Flow\Security\Account>
-	 * @ORM\OneToMany(mappedBy="party")
+	 * @ORM\OneToMany(mappedBy="party", cascade={"all"})
 	 */
 	protected $accounts;
 
